### ISSUE TITLE
create reverse proxy - Create Common Web Root for other than default Domain

### ISSUE THREAD

**Issue by harrisreddy** [IBM-Security/isam-ansible-roles#41]
the Current role for setup_common_root.yml does not have a variable to specify if we have to create the web root in a different domain. 

added a new variable to get it working in my environment, may be there are better ways to handle it. 

- name: Create Common Web Root
  isamadmin:
    appliance: "{{ inventory_hostname }}"
    username:  "{{ username }}"
    password:  "{{ password }}"
    lmi_port:  "{{ lmi_port }}"
    log:       "{{ log_level }}"
    isamuser:  "{{ create_reverseproxy_admin_id }}"
    isampwd:   "{{ create_reverseproxy_admin_pwd }}"
    isamdomain: "{{ create_pdadmin_domain }}"
    commands:
    - "object create /WebSEAL/{{common_web_root}} \"{{common_web_root_description}}\" 5 ispolicyattachable yes"
  ignore_errors: true
  when: common_web_root is defined and create_reverseproxy_admin_id is defined and create_reverseproxy_admin_pwd is defined

**Comment by ram-ibm** [comment]
Good catch - appreciate it if you can submit a pull request. The value should also be set to "Default" if it is not specified.
